rice_calculator.py

```diff
@@ -160,7 +160,11 @@ def get_rice_amount(self) -> int:
         RETURNS:
         Amount of rice wanted as an int.
         """
-        return int(questionary.text("Thank you! Now tell me how much you want to cook in grams\n").ask())
+        try:
+            return int(questionary.text("Thank you! Now tell me how much you want to cook in grams\n").ask())
+        except Exception:
+            print("Please input a number (without decimal points!)")
```

Comment:
Maybe `a correct number`

Reply:
That wouldnt answer the users question if they typed in "200grams" for example. The error message explains everything that needs to be known about what to input